docs/introduction-to-smart-contracts.rst

```diff
@@ -181,21 +181,19 @@ arguments ``from``, ``to`` and ``amount``, which makes it possible to track
 transactions.
 
 To listen for this event, you could use the following
-JavaScript code, which uses `web3.js <https://github.com/web3/web3.js/>`_ to create the ``Coin`` contract object,
+JavaScript code, which uses `ethers.js <https://github.com/ethers-io/ethers.js>`_ to create the ``coin`` contract object,
```

Comment:
Although it was already phrased like this before, maybe we could do something regarding that the text says the code "uses ether.js to **create** the coin contract object ..." but there is no line actually creating any object.
Maybe we should just change to:
```suggestion
JavaScript code, which uses `ethers.js <https://github.com/ethers-io/ethers.js>`_ to listen to ``Sent`` events broadcasted from the ``Coin`` contract ,
```

Reply:
Alternatively, we could also add a "templated" line of code which creates a ``Coin`` contract object:

```javascript
const coin = new ethers.Contract(coinAddress, abi, provider);
```

Reply:
Does that make sense, @r0qs ?